packages/picker/README.md

```diff
@@ -660,3 +688,7 @@ Use [`<sp-help-text>`](../help-text/) to add help text and error text:
 
 </sp-tab-panel>
 </sp-tabs>
+
+#### Do not use submenus
+
+A picker menu should never contain submenus, as doing so would render it in accessible. A picker's menu role is a listbox and it's menu items are listbox options, which are not allowed to have submenus.
```

Comment:
A more detailed explanation and reminder not to use submenus in a picker.

Reply:
Reinforcement 🤌 TY